tests/libdnf/sack/DnfPackageTest.hpp

```diff
@@ -0,0 +1,30 @@
+#ifndef LIBDNF_DNFPACKAGETEST_HPP
+#define LIBDNF_DNFPACKAGETEST_HPP
+
+#include <cppunit/TestCase.h>
+#include <cppunit/extensions/HelperMacros.h>
+#include <memory>
+
+#include <libdnf/sack/query.hpp>
+#include <libdnf/sack/advisory.hpp>
+
+class DnfPackageTest : public CppUnit::TestCase
+{
+    CPPUNIT_TEST_SUITE(DnfPackageTest);
+        CPPUNIT_TEST(testDnfPackageGetAdvisories);
+    CPPUNIT_TEST_SUITE_END();
+
+public:
+    void setUp() override;
+    void tearDown() override;
+
+    void testDnfPackageGetAdvisories();
+
+private:
+    DnfSack *sack;
+    HyRepo repo;
+    char* tmpdir;
```

Comment:
I believe that valgrind will complain that variables ware not correctly initialized. Please could you set for all values a default value nullptr?